jenkins-jobs/nci/templates/publisher.xml.erb

```diff
@@ -68,6 +68,15 @@
       <flatten>true</flatten>
       <doNotFingerprintArtifacts>false</doNotFingerprintArtifacts>
     </hudson.plugins.copyartifact.CopyArtifact>
+    <hudson.plugins.copyartifact.CopyArtifact plugin="copyartifact@1.35.1">
```

Comment:
would be nicer if the entire block would be loop generated for all architectures passed into the ProjectJob. this would however require changes to the ProjectJob which needs to be detached from BuilderJobBuilder and pass the architecture list into the PublisherJob so we can render that.

detaching ProjectJob kinda makes sense at this point anyway though, with multiple architectures in the picture the current meta-patching of BuilderJobBuilder becomes unsustainable. that said, this isn't blocking and the current way is fine to unbreak building. I am not sure if this will actually work as-is if there is no _bin_armhf job though (stable/user/lts)